src/js/chartAuthoring.js

```diff
@@ -185,22 +259,47 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             source: "{that}.options.dataEntryPanel",
             removeSource: true,
             target: "{that dataEntryPanel}.options"
+        },
+        {
+            source: "{that}.options.inlineEditCommonOptions",
+            removeSource: true,
+            target: "{that fluid.inlineEdit}.options"
         }]
     });
 
+    floe.chartAuthoring.highlightChange = function (that, d3Selector) {
+        var dataPlayingHighlightClass = that.options.styles.dataPlayingHighlightClass;
+
+        var selection = d3Selector;
+
+        var activeDataId = that.model.currentlyPlayingData !== null ? that.model.currentlyPlayingData.id : null;
```

Comment:
Nutty ternary - can just be 
that.model.currentlyPlayingData && that.currentlyplayingData.id
but even more directly
var activeDataId = fluid.get(model, "currentlyPlayingData.id");